rllib/utils/explained_variance.py

```diff
@@ -2,12 +2,18 @@
 from __future__ import division
 from __future__ import print_function
 
-from ray.rllib.utils import try_import_tf
+from ray.rllib.utils import try_import_tf, try_import_torch
 
 tf = try_import_tf()
+torch, nn = try_import_torch()
 
 
-def explained_variance(y, pred):
-    _, y_var = tf.nn.moments(y, axes=[0])
-    _, diff_var = tf.nn.moments(y - pred, axes=[0])
-    return tf.maximum(-1.0, 1 - (diff_var / y_var))
+def explained_variance(y, pred, backend="tf"):
```

Comment:
We've been using the term 'framework' instead of backend, would be good to be consistent with the terminology.